### PR TITLE
Start ploneintranet.api

### PR DESCRIPTION
**Pull request by mattss**
This moves the microblog api stuff into a central api (ref https://github.com/ploneintranet/ploneintranet/issues/172)

### CODE REVIEW

**Comment by ghost** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.ploneintranet.net/job/Plone.Intranet.Pull.Request.Builder/53/
Test PASSed.

File: src/ploneintranet/api/tests/test_microblog.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+
+class TestMicroblog(IntegrationTestCase):
+
+    def test_create_statusupdate(self):
+        text = 'Hello this is my update'
+        update = pi_api.microblog.create_statusupdate(text)
+        self.assertIsInstance(update, StatusUpdate)
+        self.assertEqual(update.text, text)
+
```
Comment (jcbrand) [inline src/ploneintranet/api/tests/test_microblog.py]: Shouldn't it rather be something like `api.microblog.statusupdate.get(text)` ? 
or `api.statusupdate.get(text)`? 

That seems more in line with the way plone.api does it.

Comment (mattss) [inline src/ploneintranet/api/tests/test_microblog.py]: Could indeed switch to `api.microblog.statusupdate.get` and `api.microblog.statusupdate.create`

Anyone else agree?

Comment (pysailor) [inline src/ploneintranet/api/tests/test_microblog.py]: @mattss  I agree, it would be more in line with plone.api

**Comment by mattss** [comment]
@jcbrand reworked as suggested.

**Comment by ghost** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.ploneintranet.net/job/Plone.Intranet.Pull.Request.Builder/56/
Test PASSed.

**Comment by jcbrand** [comment]
@mattss Lovely, thanks!

**Pull request merged**